x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/action/TransportStartDataFrameTransformAction.java

```diff
@@ -122,7 +120,21 @@ protected void masterOperation(StartDataFrameTransformAction.Request request,
                         transformTask,
                         persistentTaskActionListener);
                 } else {
-                    persistentTaskActionListener.onResponse(existingTask);
+                    DataFrameTransformState transformState = (DataFrameTransformState)existingTask.getState();
+                    if(transformState.getState() == DataFrameTransformTaskState.FAILED && request.isForce() == false) {
```

Comment:
So `force` is required to stop a failed job and in the case of start it means you don't have to force stop a failed job first?

Reply:
@davidkyle The idea here is that it failed, but we want to restart it after we fix the failure. Here, they can do a `_start?force=true` if it is in a `FAILED` state after corrective action has been taken. Basically, the ability to "restart" the failed task. 

Since we don't write any of the state to an index, it has to stay in cluster state for us to be able to continue from some position.